sklearn/model_selection/_search.py

```diff
@@ -355,7 +360,7 @@ def __repr__(self):
         """Simple custom repr to summarize the main info"""
         return "mean: {0:.5f}, std: {1:.5f}, params: {2}".format(
             self.mean_validation_score,
-            np.std(self.cv_validation_scores),
+            self.corrected_std,
```

Comment:
So we retain the old incorrect behaviour of accessing `grid_score[2]` while printing out the right value? That seems odd to me. I would rather either
1. Not break the old behaviour and return and print the wrong value for 2 versions (or)
2. Document this as a bug fix and both print and a

Also this change should also be done in `grid_search.py` right?